app/Policies/UserPolicy.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+namespace App\Policies;
+
+use App\Models\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+
+class UserPolicy
```

Comment:
As far as I can tell, this class is not used at all. Then we should simply just remove it instead of introducing dead code.

Reply:
Now it is :)